diff --git a/src/library/processx/R/client-lib.R b/src/library/processx/R/client-lib.R
index 9b66ae6..73db557 100644
--- a/src/library/processx/R/client-lib.R
+++ b/src/library/processx/R/client-lib.R
@@ -1,5 +1,7 @@
 client <- new.env(parent = emptyenv())
 
+if (Sys.getenv("PAK_INSTALL_DUMMY_CROSS") != "true") {
+
 local({
   ext <- .Platform$dynlib.ext
   arch <- .Platform$r_arch
@@ -44,6 +46,8 @@ local({
   }
 })
 
+}
+
 # This is really only here for testing
 
 load_client_lib <- function(client) {
diff --git a/src/library/processx/R/on-load.R b/src/library/processx/R/on-load.R
index 7282538..120c568 100644
--- a/src/library/processx/R/on-load.R
+++ b/src/library/processx/R/on-load.R
@@ -1,5 +1,7 @@
 ## nocov start
 
+if (Sys.getenv("PAK_INSTALL_DUMMY_CROSS") != "true") {
+
 .onLoad <- function(libname, pkgname) {
   ## This is needed to fix the boot time to a given value,
   ## because in a Docker container (maybe elsewhere as well?) on
@@ -35,4 +37,5 @@
   supervisor_reset()
 }
 
+}
 ## nocov end
