diff --git a/src/library/ps/src/arch/windows/process_handles.c b/src/library/ps/src/arch/windows/process_handles.c
index fd3303e3..f7fc76e2 100644
--- a/src/library/ps/src/arch/windows/process_handles.c
+++ b/src/library/ps/src/arch/windows/process_handles.c
@@ -20,7 +20,7 @@ PVOID GetLibraryProcAddress(PSTR LibraryName, PSTR ProcName) {
   return GetProcAddress(GetModuleHandleA(LibraryName), ProcName);
 }
 
-VOID ps__get_open_files_init() {
+VOID ps__get_open_files_init(void) {
   if (g_initialized == TRUE)
     return;
 
@@ -284,4 +284,3 @@ SEXP ps__get_modules(HANDLE hProcess) {
   UNPROTECT(1);
   return result;
 }
-
diff --git a/src/library/ps/src/arch/windows/process_handles.h b/src/library/ps/src/arch/windows/process_handles.h
index 61333566..fbb2539a 100644
--- a/src/library/ps/src/arch/windows/process_handles.h
+++ b/src/library/ps/src/arch/windows/process_handles.h
@@ -97,7 +97,7 @@ typedef struct _OBJECT_TYPE_INFORMATION {
 #endif
 
 PVOID GetLibraryProcAddress(PSTR LibraryName, PSTR ProcName);
-VOID ps__get_open_files_init();
+VOID ps__get_open_files_init(void);
 SEXP ps__get_open_files(long pid, HANDLE processHandle);
 DWORD ps__NtQueryObject(void);
 DWORD WINAPI ps__NtQueryObjectThread(LPVOID lpvParam);
