diff --git a/src/library/tstoml/DESCRIPTION b/src/library/tstoml/DESCRIPTION
index 80574d524..5c9ca8f22 100644
--- a/src/library/tstoml/DESCRIPTION
+++ b/src/library/tstoml/DESCRIPTION
@@ -12,7 +12,7 @@ Description: Extract and manipulate parts of TOML files without touching
 License: MIT + file LICENSE
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-Depends: R (>= 4.1.0)
+Depends: R (>= 3.5.0)
 Imports: tsitter
 Suggests: asciicast, hms, pillar, testthat (>= 3.0.0), withr
 Remotes: r-lib/tsitter
